src/BlockInfo.cpp

```diff
@@ -1054,6 +1061,7 @@ bool cBlockInfo::IsTransparent(const BLOCKTYPE Block)
 		case E_BLOCK_OAK_DOOR:
 		case E_BLOCK_OAK_FENCE_GATE:
 		case E_BLOCK_OAK_WOOD_STAIRS:
+		case E_BLOCK_OBSERVER:
```

Comment:
Does changing this actually affect redstone placement as you mentioned? If so I think we should leave this change off. Otherwise these all look good.

Reply:
It does not "cut" redstone wires (see below). So it behaves a little like a transparent block too.
![observer](https://user-images.githubusercontent.com/66266021/147486995-ee46ce10-4d56-41c4-99d3-6d791b1725d2.png)

Reply:
Also, it does not conduct redstone.
![observer](https://user-images.githubusercontent.com/66266021/147487181-a1a76241-a599-4e64-8297-1635d63d3238.png)

Reply:
Is this Cuberite or vanilla? All we care about is that they behave the same.

Reply:
It's vanilla.

Reply:
Good, seems like it does behave like a transparent block then.